src/cdk/drag-drop/drag-ref.ts

```diff
@@ -884,6 +884,7 @@ export class DragRef<T = any> {
     if (previewTemplate) {
       const viewRef = previewConfig!.viewContainer.createEmbeddedView(previewTemplate,
                                                                       previewConfig!.context);
+      viewRef.detectChanges();
```

Comment:
Rather than directly calling `detectChanges`, couldn't we instead defer measuring until Angular completes the change detection it needs to do anyway?

Reply:
Almost none of the drag&drop events trigger change detection so we can't really depend on it since it can happen at any time for other reason. This only detects on a small chunk of the tree anyway.